images/build/Dockerfile

```diff
@@ -115,6 +115,18 @@ RUN curl -sL https://git.io/n-install | bash -s -- -ny - \
  && ~/n/bin/n -d 10.15.2 \
  && mv /usr/local/n/versions/node /opt/nodejs \
  && rm -rf /usr/local/n ~/n
+RUN set -e \
+ && for ver in `ls /opt/nodejs`; do \
```

Comment:
So in the build image, we would be looking into _every_ installed Node version and then see if the npm version that comes bundled with it has a major version of `6` and if yes, will upgrade them to `6.9.0`. This restriction is so that we do not make 'major' upgrade which could possibly break apps.